src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -631,4 +642,125 @@ public static void updateConfigErrorMessage(Config result, String key, String ms
       }
     }
   }
+
+  /**
+   * get schema with its subject being [topicName]-value
```

Comment:
schema is stored in the schema registry in terms of subject, in our case the subject name could either be "<topicName>-value" or "<topicName>-key". These names are set by the producer (or schema registry client) automatically. 

We don't need to do anything with "<topicName>-key" because the content of the key will only be present in the RECORD_METADATA, so this method will only fetch the schema from "<topicName>-value"